src/openforms/template/tests/test_validators.py

```diff
@@ -1,13 +1,18 @@
 from django.core.exceptions import ValidationError
+from django.template import engines
+from django.template.backends.django import DjangoTemplates
 from django.test import SimpleTestCase, override_settings
 from django.utils.translation import gettext as _
 
 from ..validators import DjangoTemplateValidator
 
+django_backend = engines.all()[0]
+assert isinstance(django_backend, DjangoTemplates)
```

Comment:
we are testing that the `DjangoTemplateValidator` works correctly in the "generic" sense, without testing any particular backend.

The `{% csrf_token %}` and `{% load i18n %}` template constructs are core django tokens that will not break if we have a bug in our own tags, or won't be affected by unrelated changes (e.g. changing the `{% form_summary %}` tag syntax would break these tests but not because the validator broke).

Before this patch, the `DjangoTemplateValidator` relied on `Template`, which under the hood uses `Engine.get_default()`, which just grabs the first Django templates enginge/configuration from `settings.TEMPLATES`. However, this patch defaults to using our own sandbox as backend for validation (secure defaults), which doesn't allow usage of these built-in django tags.

So, two options to fix these tests:

* replace the tags in there with sandbox-allowed tags
* force the old validator behaviour by using the backend from `Engine.get_default()`

I opted for the second so that the actual template doesn't matter/won't cause issues while still testing the correct validator behaviour. Obtaining this django template instance is a bit convoluted and there's a safe guard in case at some point our first entry in `settings.TEMPLATES` is no longer DTL (but Jinja2 for example). Unlikely to happen, but just in case, this can save some debugging time.

> If not, I'd import the one we expect and assert that it's the correct one

that's essentially what I'm doing - instantiating without duplicating all the settings from settings.TEMPLATES manually is... surprisingly hard and fragile if/when django upgrades change template internals.

The `APP_DIRS` flip is unrelated to this, that's another cleanup of something that used to be required in the past, but what Django now does automatically.